### PR TITLE
Fix to release1.4 flaky and few tags updates

### PR DESCRIPTION
**Pull request by praveen4g0**

### CODE REVIEW

File: pkg/pipelines/pipelines.go [hunk 1]
```diff
@@ -249,7 +249,7 @@ func AssertForNoNewPipelineRunCreation(c *clients.Clients, namespace string) {
 		}
 	}()
 	time.Sleep(1 * time.Minute)
-	if count != expectedCount {
+	if count >= expectedCount {
```
Comment (praveen4g0) [inline pkg/pipelines/pipelines.go:252]: condition should be less than :wink:  my bad

**Comment by praveen4g0** [comment]
tested! and it's working!

**Review by ppitonak — APPROVED** [review]

**Comment by ppitonak** [comment]
Please rebase manually

**Comment by praveen4g0** [comment]
rebased!

**Pull request merged**